yb-voyager/cmd/importData.go

```diff
@@ -259,11 +263,11 @@ func importData(importFileTasks []*ImportFileTask) {
 	if !dbzm.IsDebeziumForDataExport(exportDir) {
 		executePostImportDataSqls()
 	} else {
-		status, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
-		if err != nil {
-			utils.ErrExit("failed to read export status for restore sequences: %s", err)
-		}
 		if changeStreamingIsEnabled(importType) {
+			status, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
+			if err != nil {
+				utils.ErrExit("failed to read export status for restore sequences: %s", err)
+			}
```

Comment:
BUG: this code block needs to be executed _AFTER_ `streamChanges()` and _BEFORE_ `RestoreSequences()`.